src/common/feedGenerator.ts

```diff
@@ -536,9 +537,19 @@ export const sourceFeedBuilder = (
 ): SelectQueryBuilder<Post> => {
   builder.andWhere(`${alias}.sourceId = :sourceId`, { sourceId });
 
-  if (sourceId === 'community') {
-    builder.andWhere(`${alias}.banned = false`);
-  }
+  builder.andWhere(
+    new Brackets((qb) => {
+      qb.where(`${alias}.authorId = :userId OR ${alias}.scoutId = :userId`, {
+        userId: ctx.userId,
+      }).orWhere(whereVordrFilter(alias));
+
+      if (sourceId === 'community') {
+        qb.andWhere(`${alias}.banned = false`);
+      }
+
+      return qb;
```

Comment:
Does this not need to be outside of bracket since the bracket itself is `OR`?

Reply:
It works when it's inside the bracket

Reply:
I can test it outside to see if it also works

Reply:
that last bracket then is `"authorId" = :userId OR "scoutId" = :userId OR (VORD) AND "banned" = false`

and it should be `("authorId" = :userId OR "scoutId" = :userId OR (VORD)) AND "banned" = false` no?

Reply:
No, when it's outside the brackets, I don't see the banned posts that I am the author of that is banned by vordr

Reply:
synced offline